src/components/Footer/Footer.jsx

```diff
@@ -4,15 +4,130 @@ import React from 'react';
 import Box from '@mui/material/Box';
 import Button from '@mui/material/Button';
 import Container from '@mui/material/Container';
-import Grid from '@mui/material/Grid';
+import Divider from '@mui/material/Divider';
 import Link from '@mui/material/Link';
+import Stack from '@mui/material/Stack';
 import Typography from '@mui/material/Typography';
+import useMediaQuery from '@mui/material/useMediaQuery';
 import { useTheme } from '@mui/material/styles';
 // Material Icons Imports
 import TwitterIcon from '@mui/icons-material/Twitter';
 import FacebookIcon from '@mui/icons-material/Facebook';
 import InstagramIcon from '@mui/icons-material/Instagram';
 
+// top section of footer
+const RenderCallToActionSection = ({ isReallySmallScreen }) => (
+  <Stack width={isReallySmallScreen ? 1 : 3 / 5} alignItems="center" justifyContent="center">
+    <Typography variant="h5" color="tertiary.main">
+      Want to partner with PASS?
+    </Typography>
+    <Typography variant="body1" color="#fff" sx={{ width: 3 / 4 }}>
+      If your organization is interested in partnering with PASS and would like to discuss further,
+      contact us below.
+    </Typography>
+    <Button variant="contained" color="secondary" sx={{ my: '1rem', width: 1 / 2 }}>
+      Proposal
+    </Button>
+  </Stack>
+);
+
+// middle section of footer
+const RenderCompanyInfoSection = ({ isReallySmallScreen }) => {
+  const socialLinks = [
```

Comment:
Can have socialLinks be called outside of component so the object doesn't have to be called each time component needs to be rendered